test/e2e/storage/persistent_volumes.go

```diff
@@ -300,4 +300,53 @@ var _ = utils.SIGDescribe("PersistentVolumes", func() {
 			})
 		})
 	})
+
+	Describe("Default StorageClass", func() {
+		Context("pods that use multiple volumes", func() {
+			It("should be reschedulable", func() {
```

Comment:
does this test actually accurately reproduce the error?

Reply:
no, the issue is very incredibly difficult to reproduce and I have not found a consistent way to get into the error state